lib/matplotlib/mpl-data/matplotlibrc

```diff
@@ -424,8 +424,17 @@
 #axes.autolimit_mode: data  # If "data", use axes.xmargin and axes.ymargin as is.
                             # If "round_numbers", after application of margins, axis
                             # limits are further expanded to the nearest "round" number.
-#polaraxes.grid: True  # display grid on polar axes
-#axes3d.grid:    True  # display grid on 3D axes
+#axes.adjustable: box       # {box, adjustable}
+#axes.anchor:    C          # {C, E, N, S, W, NE, NW, SE, SW} or two-tuple of floats
+#axes.aspect:    auto       # {equal, auto} or a number
+
+#polaraxes.grid: True       # display grid on polar axes
+
+#axes3d.grid:    True       # display grid on 3D axes
+#axes3d.adjustable: box     # {box, adjustable}
+#axes3d.anchor:     C       # {C, E, N, S, W, NE, NW, SE, SW} or two-tuple of floats
```

Comment:
Why not inherit these from `axes.adjustable` or `axes.anchor`? A lot of other 3D properties fall back on this without setting an explicit 3D param.

Reply:
https://github.com/matplotlib/matplotlib/issues/25641 :-) 

One can for sure discuss which can be the same and which are beneficial to have separately.

Reply:
#25641 is definitely interesting. 

For this one, as defaults, maybe defaulting to what is done is 2d is useful?

Reply:
Sorry, I do not get it. Are you saying that if the 3D-version of the RCparam is not changed, but the 2D-version is, it should use the 2D-version? How do we know if the style-sheet has set the 3D-version to the default value or not? I may be missing something, but I didn't think it was possible to query if the param comes from the style sheet or if the default is used?

Also, wouldn't that be a bit confusing? Now we add double parameters that are also physically located quite nearby, so I guess if someone want to change the 2D-case, chances are that they will notice the 3D-case as well?

Then it may very well be that it is not worthwhile to have separate 2D and 3D versions for all parameters introduced here. `aspect` and ´adjustable` seems to make sense to have separately, which leaves `anchor`. That was basically just added because it ended up nearby in the code and I realized that it could be done.

Reply:
OK, then I understand. Yes, it is not a problem as such, more a question of which parameters we would like to separate.

Reply:
Yes. But I guess that's a part of a bigger conversation about the role of rcparams itself. For now, I don't mind having all 3 separate parameters either.